docs/source/deployment/nodes_grouping.md

```diff
@@ -0,0 +1,107 @@
+- With the visualisation in Kedro Viz, you can switch to see different pipeline in a more isolated view
+<br>
+![Switching between different pipelines in Kedro Viz](../meta/images/kedro_viz_switching_pipeline.gif)
+
+### What doesn't work with Kedro
+- If you need to create custom node groupings that are different with your existing pipelines, creating new pipelines for them may not be convenient. Instead, you can use alternative grouping methods such as tags or namespaces.
+- You cannot execute more than one pipeline in a single step because the kedro run --pipeline command allows running one pipeline at a time.
+- You can switch between different pipelines in Kedro Viz, but the visualisation does not support collapsing or expanding pipelines within Kedro Viz.
+
+### Best used when
+- Your project is structured in a way that requires executing pipelines separately in the deployment environment.
+
+### Not to use when
+- You need to run more than one pipeline together.
+- You want to use the expand or collapse functionality in Kedro Viz
+
+### How to use
+- Run using:
+  ```bash
+  kedro run --pipeline=<your_pipeline_name>
+  ```
+- More information: [Run a pipeline by name](https://docs.kedro.org/en/stable/nodes_and_pipelines/run_a_pipeline.html#run-a-pipeline-by-name)
+
+---
+
+## Tags
+
+### What works with Kedro
+- You can tag individual nodes or the entire pipeline, allowing flexible execution of specific sections without modifying the pipeline structure.
+- Kedro-Viz provides a clear visualisation of tagged nodes, making it easier to understand.
+<br>
+![Filters Panel in Kedro Viz](../meta/images/kedro_viz_filters_tags.png)
+
+### What doesn't work with Kedro
+- Nodes with the same tag can exist in different pipelines, making debugging and maintaining the codebase more challenging.
+- Executing more than one tag in a single step isn't always possible, and sometimes it will fail to run if a part of pipeline is not populated before at the time of execution.
+
+### Best used when
+- You need to run specific nodes that don’t belong to the same pipeline.
+- You want to rerun a subset of nodes in a large pipeline.
+
+### Not to use when
+- The tagged nodes have strong dependencies, which might cause execution failures.
+- Tags are not hierarchical, so tracking groups of nodes can become difficult.
+- Tags do not enforce structure like pipelines or namespaces.
+
+### How to use
+- Run using:
+  ```bash
+  kedro run --tags=<your_tag_name>
+  ```
+- More information: [How to tag a node](https://docs.kedro.org/en/stable/nodes_and_pipelines/nodes.html#how-to-tag-a-node)
+
+---
+
+## Namespaces
+
+### What works with Kedro
+- Namespaces allow you to group nodes, ensuring clear dependencies and separation within a pipeline while maintaining a consistent structure.
+- As the same as pipelines or tags, you can enable selective execution using namespaces.
+- Namespaces improve visualisation in Kedro-Viz.
```

Comment:
Maybe "Kedro Viz supports expand and collapsing namespaces as nested pipelines" or sth like that?